src/main.html

```diff
@@ -402,6 +402,21 @@ <h3 i18n="warningTitle"></h3>
         </div>
     </dialog>
 
+    <dialog class="dialogResetToCustomDefaults">
+        <h3 i18n="noticeTitle"></h3>
+        <div class="content">
+            <div class="dialogResetToCustomDefaults-content" style="margin-top: 10px margin-bottom: 10px"></div>
```

Comment:
Is better to not include styles at the HTML code. Move it to a css.

Reply:
the ship on classes vs id's probably sailed years ago, but that class really looks like what a person would name an id. 

slightly different CSS/jQuery selector
class: `.dialogResetToCustomDefaults-content` 
id: `#dialogResetToCustomDefaults-contents`

Reply:
Heh, good points. I just copied what I already had without contemplating if it was good or not.

Will fix.